masschallenge/django-accelerator
# [AC-9622]: [Program Interest]: Create New Industry field on Program Admin

[AC-9622](https://masschallenge.atlassian.net/browse/AC-9622)

How to test
- Check out branch `AC-9622`
- Run `make migrate`
- Ensure that the def of done according to [AC-9622](https://masschallenge.atlassian.net/browse/AC-9622) are met

[Sibling PR](https://github.com/masschallenge/accelerate/pull/3319)
